SQL/New_patches/2024-11-29-Dictionary-Field-Permissions.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE `parameter_type_permissions_rel` (
+  `ParameterTypeID` int(10) unsigned NOT NULL,
```

Comment:
There is no ParameterTypeID in the dictionary module. This needs to store Module and Fieldname.